lib/Fencer/Server.hs

```diff
@@ -33,19 +34,25 @@ import qualified Fencer.Proto as Proto
 -- | Run the gRPC server serving ratelimit requests.
 --
 -- TODO: fail if the port is taken? or does it fail already?
-runServer :: Logger -> AppState -> IO ()
-runServer logger appState = do
+runServerWithPort :: Port -> Logger -> AppState -> IO ()
+runServerWithPort (Port port) logger appState = do
     let handlers = Proto.RateLimitService
             { Proto.rateLimitServiceShouldRateLimit = shouldRateLimit logger appState
             }
     let options = Grpc.defaultServiceOptions
             { Grpc.serverHost = "0.0.0.0"
+            , Grpc.serverPort = fromIntegral port
               -- TODO: set the logger
             }
     Logger.info logger $
         Logger.msg (Logger.val "Starting gRPC server at 0.0.0.0:50051")
     Proto.rateLimitServiceServer handlers options
 
+-- | Run the gRPC server serving ratelimit requests on the default
+-- 50051 port.
+runServer :: Logger -> AppState -> IO ()
+runServer = runServerWithPort (Port 50051)
```

Comment:
`50051` is mentioned here and in `lib/Fencer/Settings.hs`. I'd make a single named constant (`defaultPort` or the like).

Reply:
Done.